src/bundle/Resources/public/scss/_custom.scss

```diff
@@ -104,3 +104,6 @@ $link-color: $ez-color-hyperlink;
 
 // Body
 $body-bg: $ez-ground-base-pale;
+
+// Transition function
+$admin-transition: cubic-bezier(.25,.8,.25,1);
```

Comment:
Here are bootstrap variables. I would suggest to put it on top of the file and use prefixed name `$ez-admin-transition`